pkg/loop/telem.go

```diff
@@ -66,15 +67,25 @@ func SetupTracing(config TracingConfig) error {
 	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
Is this still reasonable going forward as we continue to add things here? I'm concerned that it is arbitrary and we have no escape hatch. Actually, was it only added when we had a blocking connection? Could we just drop it?
```suggestion
```

Reply:
We can drop it, good call